backend/inspirehep/submissions/views.py

```diff
@@ -259,12 +263,31 @@ def create_ticket(self, record, rt_template):
         )
 
 
+class ExperimentSubmissionsResource(BaseSubmissionsResource):
+    def load_data_from_request(self):
+        return experiment_loader_v1()
+
+    def post(self):
+        """Adds new experiment record"""
+
+        request_data = self.load_data_from_request()
+        data = {
+            "_collections": ["Experiments"],
+            "$schema": "https://inspirehep.net/schemas/records/experiments.json",
```

Comment:
I would put that in marshmallow, also if you'll have a proper serializer `self.load_data_from_request()` should prepare your data in a correct format for creating a record.

Reply:
hmm, but how do I add the ```$schema``` there? $ makes it a bit tricky or do you mean the whole dict?

Reply:
you can have a method for building this data dict (which won't use builder).

Reply:
Ah okay. Now I understand. Thank you